BSPConversionLib/Source/EntityConverter.cs

```diff
@@ -137,6 +143,88 @@ private HashSet<string> GetGiveTargets()
 			return targets;
 		}
 
+		private void ConvertFuncDoor(Entity entity)
+		{
+			GetMoveDir(entity);
+
+			if (float.TryParse(entity["health"], out var health))
+			{
+				entity.ClassName = "func_button"; // Health is obsolete on func_door, maybe fix in engine and update this
+			}
+		}
+
+		private void ConvertFuncButton(Entity entity)
+		{
+			GetMoveDir(entity);
+
+			GetButtonFlags(entity);
+
+			if (entity["wait"] == "-1")
```

Comment:
What is this code doing exactly? -1 wait means "no wait" in Q3 and "don't reset" in Source, right?
I'd suggest adding a comment that explains the code.